plugins/inara.py

```diff
@@ -669,10 +669,13 @@ def journal_entry(  # noqa: C901, CCR001
                     to_send = {
                         'starsystemName':       system,
                         'stationName':          entry['Name'],
-                        'marketID':             entry['MarketID'],
                         'starsystemBodyName':   entry['BodyName'],
                         'starsystemBodyCoords': [entry['Latitude'], entry['Longitude']]
                     }
+                    # Not present on, e.g. Ancient Ruins
+                    if market_id := entry.get('MarketID') is not None:
```

Comment:
This may cause market_id to be a bool?

Reply:
Yes, yes it does.

f/x: Ath goes off grumbling about Python operator precedence